config-service/build.gradle.kts

```diff
@@ -14,6 +14,7 @@ plugins {
   id("org.hypertrace.docker-publish-plugin")
   id("org.hypertrace.integration-test-plugin")
   id("org.hypertrace.jacoco-report-plugin")
+  id("org.hypertrace.publish-plugin")
```

Comment:
The main config service jar should not be published - let me look what our options are here

Reply:
Raised a pr into this branch - https://github.com/hypertrace/config-service/pull/27 . It moves the test fixtures to the api package, which is a better home for them.